app/presenters/users/sessions/reset_password.rb

```diff
@@ -1,27 +1,27 @@
 module Users
   module Sessions
     class ResetPassword < ApplicationPresenter
-      def initialize(view_context, token)
+      def initialize(view_context, token, user)
         @token = token
+        @user = user
         super(view_context)
       end
 
       def page_title
-        "Reset Password | #{school_name}"
+        "Reset Password | #{current_school.name}"
```

Comment:
It's a good opportunity to move this to `en.yml` like many other pages.

Reply:
I am not sure, if I have placed the string in `en.yml` at the right place.